src/special/build_runner.zig

```diff
@@ -243,9 +243,7 @@ fn processModules(
     for (modules.keys(), modules.values()) |name, mod| {
         const path = mod.builder.pathFromRoot(mod.source_file.getPath(mod.builder));
 
-        const already_added = try packages.addPackage(name, path);
-        // if the package has already been added short circuit here or recursive modules will ruin us
-        if (already_added) return;
+        _ = try packages.addPackage(name, path);
```

Comment:
```suggestion
        if (try packages.addPackage(name, path)) continue;
```

Reply:
How did I not realize that I'm in a for loop. My bad.